src/test/java/tests/auth/UserAuthServiceTest.java

```diff
@@ -122,4 +123,19 @@ public void userInvalidateToken() {
                 .body("httpStatus", equalTo("OK"))
                 .body("isSuccess", equalTo(true));
     }
+    @Test()
+    public void testUserInvalidRefreshTokenForAccessTokenCreation(){
+        Token token = Helper.getUserToken(userCredentials);
+        RefreshToken refreshToken = new RefreshToken();
+        refreshToken.setRefreshToken(token.getRefreshToken());
+        UserAuthEndpoints.userInvalidateToken(token.getAccessToken(), refreshToken);
+        Response response=InstitutionAuthEndpoints.adminTokenRefresh(refreshToken);
+        response.then()
+                .statusCode(401)
+                .contentType("application/json")
+                .body("time", notNullValue())
+                .body("httpStatus", equalTo("UNAUTHORIZED"))
+                .body("header", equalTo("AUTH ERROR"))
+                .body("isSuccess", equalTo(false));
```

Comment:
Bu tarz birbirini tekrar eden durumlar için ortak bir yapı kurulabilir, örneğin AysErrorResponse gibi bir sınıf açılarak içerisine UNAUTHORIZED objesi eklenerek çağrılabilir. Ek olarak `"time"`, `"header"` gibi String ifadeler de birbirini tekrar ediyor bunlar da ortaklaştırılabilir.

Reply:
Bunu bu issue icersinde mi yapalım yoksa ayrı bir issue acarak mi?

Reply:
Eğer mümkünse burada örnek bir yapı oluşturmak iyi olabilir, uygulama genelini refactor etmek ayrı bir issue içerisinde yapılır. Burayı düzeltmeden ilerlersek bu sefer uygulamada çok fazla refactor edilmesi gereken yer olmaya başlar. Bu da çok uzun sürmesine sebep olabilir.

Reply:
Haklisin, bu sekilde düzenlemeye calisacigim.